### DISCUSSION TITLE
Tabキーでのインデント時にnoteが使えない

### DISCUSSION THREAD

**Discussion by lemonade-37** [increments/qiita-discussions#888] [要望・アイデア・不具合]
```
:::note
:::
```

`1. 2. 3. `などのインデックスをつけた時に、各番号内の内容を記載する時に、
2行目以降はTabキーでインデントをつけて記載します。
インデント時にnote機能を使う際だけバグがあります。
閉じる側の`:`が1つだけ不自然に表示されます。

**Comment by tanaka-asami415** [comment]
@satou-haruka-37 
ご報告ありがとうございます🙇
調査の上、修正いたします！

**Reply by lemonade-37** [reply]
ありがとうございます！！🙇‍♀️
